sirepo/template/radia_tk.py

```diff
@@ -152,6 +155,10 @@ def field_integral(g_id, f_type, p1, p2):
     return radia.FldInt(g_id, 'inf', f_type, p1, p2)
 
 
+def free_symmetries(g_id):
+    return radia.ObjDpl(g_id, 'FreeSym->True')
+
+
```

Comment:
the line below isinstance is redundant. PKDict is an instance of dict

Reply:
f853973